mscore/musescore.cpp

```diff
@@ -6018,6 +6018,33 @@ void MuseScore::cmd(QAction* a, const QString& cmd)
                   cv->setMag(MagIdx::MAG_100, 1.0);
             setMag(1.0);
             }
+      // 2019.12.25: [Page-Width] as shortcut command
```

Comment:
The date is definitely not needed. But I assume this entire comment is not needed too.

Reply:
Ironically, I think I added that comment to verify that there was a difference at first when trying to get this thing working on here, and now I have to remove it. Haha.

Reply:
I see xD